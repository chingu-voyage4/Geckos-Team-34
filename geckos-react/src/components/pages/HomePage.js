import React from 'react';

const HomePage = () => {
  return (
    <div className="ui container">
      <h1>Home Page</h1>
    </div>
  );
};

export default HomePage;