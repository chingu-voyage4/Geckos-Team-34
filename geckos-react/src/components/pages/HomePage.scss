.home-page {
  background-color: green;
}
