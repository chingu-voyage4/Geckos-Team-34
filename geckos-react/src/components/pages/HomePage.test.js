import React from 'react';
import { shallow } from 'enzyme';

import HomePage from './HomePage';

describe('<HomePage />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<HomePage />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.length).toBe(1);
  });

  it('should render h1 title', () => {
    const elem = wrapper.find('h1');
    expect(elem.length).toBe(1);
    expect(elem.text()).toBe('Home Page');
  });
});