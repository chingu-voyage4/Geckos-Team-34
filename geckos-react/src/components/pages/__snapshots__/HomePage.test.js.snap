// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render correctly 1`] = `
<div
  className="ui container"
>
  <h1>
    Home Page
  </h1>
</div>
`;
